src/Components/Header/Header.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useEffect, useRef, useState } from 'react';
+import cn from 'classnames';
+
+import { addTodos } from '../../api/todos';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[];
+  isDisableInput: boolean;
+  isCompletedTodos: boolean,
+  setTodos: React.Dispatch<React.SetStateAction<Todo[]>>;
+  setIsDisableInput: (value: boolean) => void;
+  setTempTodo: (todo: Todo | null) => void;
+  makeAllTodosCompleted: () => void;
+  showErrorMessage: (message: string) => void;
+};
+
+export const Header: React.FC<Props> = ({
+  todos,
+  setTodos,
+  isDisableInput,
+  isCompletedTodos,
+  setIsDisableInput,
+  setTempTodo,
+  makeAllTodosCompleted,
+  showErrorMessage,
+}) => {
+  const [titleTodo, setTitleTodo] = useState('');
+  const textInput = useRef<HTMLInputElement | null>(null);
+  const isUnableAddTodo = useRef(true);
+
+  useEffect(() => {
+    textInput.current?.focus();
+  }, [isDisableInput]);
+
+  const onSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    const newTodo = {
```

Comment:
If you write your constant with userId in another file 'helpers' you can export this constant instead ```11853```. I think it's will be better